src/arch/arm/64/kernel/vspace.c

```diff
@@ -1071,45 +1081,36 @@ pude_t *pageDirectoryMapped(asid_t asid, vptr_t vaddr, pde_t *pd)
 
 #ifdef CONFIG_ARM_HYPERVISOR_SUPPORT
 
-static void invalidateASID(asid_t asid)
+static asid_map_t *findMapRefForASID(asid_t asid)
 {
-    asid_pool_t *asidPool;
+    asid_pool_t        *poolPtr;
```

Comment:
is the additional indent intentional?

Reply:
It came in from the cherry-pick. I assumed the style tool would fix it but it seems not.